src/models/multiBarChart.js

```diff
@@ -179,11 +179,6 @@ nv.models.multiBarChart = function() {
             if (showLegend) {
                 legend.width(availableWidth - controlWidth());
 
-                if (multibar.barColor())
-                    data.forEach(function(series,i) {
-                        series.color = d3.rgb('#ccc').darker(i * 1.5).toString();
-                    });
```

Comment:
The one change though is that (on the last image there, where barColor is used) the legend uses a color instead of shades of gray to denote each series, before it used shades of gray... the shades of blue there seems confusing given the colors of the bars...